pkg/images/enricher/enricher_impl_test.go

```diff
@@ -486,7 +486,7 @@ func TestEnricherFlow(t *testing.T) {
 			require.NoError(t, err)
 			assert.Equal(t, c.result, result)
 
-			assert.Equal(t, c.fsr, fsr)
+			assert.Equal(t, c.fsr, fsr) //nolint:donotcompareproto // false possitive
```

Comment:
I saw there was originally a comment for the Scanner team to review this, but now it's gone. Still need a review?